app/src/main/java/com/isc/hermes/model/User.java

```diff
@@ -0,0 +1,202 @@
+package com.isc.hermes.model;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+/**
+ * The `User` class represents a user in the application. It stores information about the user, such as their full name,
+ * email, username, profile image path, user type, and ID.
+ */
+public class User implements Parcelable {
+    private String fullName;
+    private String email;
+    private String userName;
+    private String pathImageUser;
+    private String typeUser;
+    private String id;
+
+    /**
+     * Constructs a User object with the specified email, path to the user's image, and ID.
+     *
+     * @param email The email of the user.
+     * @param pathImageUser The path to the user's image.
+     * @param id The ID of the user.
+     */
+    public User(String email, String pathImageUser, String id) {
+        this.email = email;
+        this.pathImageUser = pathImageUser;
+        this.id = id;
+    }
+
+    /**
+     * Constructs a User object from a Parcel.
+     *
+     * @param in The Parcel from which to read the User object.
+     */
+    protected User(Parcel in) {
+        fullName = in.readString();
+        email = in.readString();
+        userName = in.readString();
+        pathImageUser = in.readString();
+        typeUser = in.readString();
+        id = in.readString();
+    }
+
+    /**
+     * Returns the first word in a string of words.
+     *
+     * @param words The string of words.
+     * @return The first word in the string.
+     */
+    private String getFirstWord(String words) {
+        String[] wordsArray = words.split(" ");
+        String expectedWord = "";
+
+        if (wordsArray.length > 0) expectedWord = wordsArray[0];
+        return expectedWord;
+    }
```

Comment:
This method should be in another class. Like Utils for example.

Reply:
Done